var/spack/repos/builtin/packages/mumps/package.py

```diff
@@ -323,7 +323,10 @@ def install(self, spec, prefix):
             ['d', '+double'], ['z', '+complex+double']]
         for ltr, v in letters_variants:
             if v in spec:
```

Comment:
`# Build fails in parallel`
The change (no more `make dexamples`) is due to some work on the makefiles to allow parallel build (`make -j all`) according to Mumps developers.

Reply:
Yeah but I think it is simpler to keep it the way it currently is to support older versions.